tsconfig.json

```diff
@@ -16,4 +16,11 @@
     "typeRoots": ["node_modules/@types"],
     "lib": ["es2017", "dom"]
   }
+
```

Comment:
Do we want this here commented?

Reply:
Yes, we want that to enable this later. 
At the moment there are much error with this new Angular linter in our project